HAR-6770 Disabloi valinnat jos vain yksi vaihtoehto

Kauko opened the pull request.

Kauko committed:
Lisää lomakkeelle :valitse-ainoa? optio

:valitse-ainoa? voi antaa :valinta tyyppisille kentille.
Jos true, resetoi dataan ainoan :valinta-vaihtoehdon, ja
kutsuu nayta-arvo, eikä tee-kentta funktiota.

Kauko committed:
Älä anna muokata sopimusta, toimenpidettä, jos vaihtoehtoja vain yksi

Kauko committed:
Merge branch 'develop' into HAR-6770

Kauko committed:
Merge branch 'develop' into HAR-6770

Kauko committed:
Merge branch 'develop' into HAR-6770

erno commented on src/cljs/harja/ui/lomake.cljs:
```diff
@@ -218,8 +218,17 @@ Ryhmien otsikot lisätään väliin Otsikko record tyyppinä."
                  (if muokattava?
-                   (do (have #(contains? % :tyyppi) s)
-                       [tee-kentta (assoc s :lomake? true) arvo])
+                   (if (and (= :valinta (:tyyppi s))
```
Tämä valitse-ainoa? avain voitaisiin destrukturoida tuosta parametrimapista heti alussa kuten muutkin, niin optiot olisi nähtävillä kaikki samassa paikassa.

Kauko committed:
:valitse-ainoa? optio kunnioittaa paremmin :valinta-kentan muita optioita

Aiempi ratkaisu ei toiminut, jos yritettiin käyttää :valinta-nayta
ja :valinta-arvo optioita.

Kauko committed:
Destrukturoi parametreja

erno committed:
Merge branch 'develop' into HAR-6770
